src/components/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,41 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import React from 'react';
+
+import { ComponentPropsWithoutRef } from 'react';
+
+export interface TabsProps extends ComponentPropsWithoutRef<'div'> {
```

Comment:
maybe it's "ToogleProps"?

Reply:
Yes! I noticed that! It's already updated!

Reply:
Copy & Paste... you know... 😆